packages/ui/__tests__/components/organisms/DropdownCardSwitch.test.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import {View} from 'react-native';
+import {shallow} from 'enzyme';
+import {DropdownCardSwitch, DropdownCard, Text} from '@axelor/aos-mobile-ui';
+import {getGlobalStyles} from '../../tools';
+
+describe('DropdownCardSwitch', () => {
+  const dropdownItems = [
+    {key: 1, title: 'Item 1', childrenComp: <Text>Content 1</Text>},
+    {key: 2, title: 'Item 2', childrenComp: <Text>Content 2</Text>},
+  ];
+
+  it('renders without crashing', () => {
+    const wrapper = shallow(
+      <DropdownCardSwitch dropdownItems={dropdownItems} />,
+    );
+
+    expect(wrapper.exists()).toBe(true);
+  });
+
+  it('renders correctly with dropdown items', () => {
+    const wrapper = shallow(
+      <DropdownCardSwitch dropdownItems={dropdownItems} />,
+    );
+    expect(wrapper.find(DropdownCard)).toHaveLength(dropdownItems.length);
+  });
```

Comment:
At space and check that props are correctly given to each DropdownCard (title, children)
(check StarScore, RadioSelect or MultiValuePicker for examples)